docs/src/components/aptitudes/aptitudeData.tsx

```diff
@@ -0,0 +1,440 @@
+import React from 'react';
+
+export interface LDKLinks {
+  node: string;
+  go: string;
+  dotnet: string;
+}
+
+export type LDKLinkActive = Partial<LDKLinks>;
+
+export interface IAptitudeData {
+  name: string;
+  description: React.ReactNode;
+  capabilities: ICapabilityData[];
+  links?: LDKLinkActive;
+}
+
+export interface ICapabilityData {
+  name: string;
+  description: React.ReactNode;
+  links?: LDKLinkActive;
+}
+
+export const aptitudes: { [sensor: string]: IAptitudeData } = {
```

Comment:
Right now this data is consumed both by the `gatsby-node.ts` file where we build the individual pages, as well as in the individual sensor pages. I think the ideal approach would be to place all of the data within the GraphQL DB and then consume it from there, but we'd be losing the ability to provide elements.

The long term solution is, IMO to add MDX support and then figure out how to structure the data such that we can distinguish in the aptitude file the data for the individual capabilites (or maybe one file per capability in addition to the aptitude file)